### ISSUE TITLE
How is XLM-R training batch size getting 8192

### ISSUE THREAD

**Issue by shenfe** [pytorch/fairseq#2798]
As the [paper](https://arxiv.org/pdf/1911.02116.pdf) says, 

> We use the multi- lingual MLM loss and train our XLM-R model for 1.5 Million updates on five-hundred 32GB Nvidia V100 GPUs with a batch size of 8192. 

But the paper doesn't tell what does 8192 mean?
- the mini-batch size on each **single** gpu (if so, how's it possible? splitting the computation upon each batch?)
- the size of a batch (with some gradient accumulation steps in it) on each **single** gpu
- the total size of a batch on **all** gpus at the same time (but 8192 / 500 != 0 ...)
- any else?

**Comment by myleott** [comment]
That's a typo in the paper. It was trained on 512 GPUs with a total batch size of 8192 sequences (16 sequences per GPU).
